tests/evergreen/test_api.py

```diff
@@ -341,6 +342,27 @@ def test_test_stats_by_project(self, mocked_api):
             url=expected_url, params=expected_params, timeout=None, data=None, method="GET"
         )
 
+    def test_test_stats_by_project_with_requester(self, mocked_api):
+        after_date = "2019-01-01"
+        before_date = "2019-02-01"
+        expected_url = mocked_api._create_url("/projects/project_id/test_stats")
+        expected_params = {
+            "after_date": after_date,
+            "before_date": before_date,
+            "requesters": "patch",
```

Comment:
shouldn't this value be `patch_request` instead of `patch`?

Reply:
No, "patch" is correct here. It is a very-annoying part of the Evergreen API. The "*stats" endpoints [use different values](https://github.com/evergreen-ci/evergreen/wiki/REST-V2-Usage#fetch-the-test-stats-for-a-project) for Requester than [other endpoints](https://github.com/evergreen-ci/evergreen/wiki/REST-V2-Usage#version).